src/cloud/project/privatelink-service.md

```diff
@@ -131,22 +131,27 @@ To test the connection to the VPC endpoint service:
    curl telnet://<endpoint-service-dns-url>:<port>/ -vvv
    ```
 
-   For example:
+   Example:
 
-   ```bash
-   curl telnet://vpce-004678b7ff06dfxxx-qmmwuxxx.vpce-svc-0aca4ec0ef530xxxx.us-east-1.vpce.amazonaws.com:8443/ -vvv
+   ```terminal
+   $ curl -v telnet://vpce-007ffnb9qkcnjgult-yfhmywqh.vpce-svc-083cqvm2ta3rxqat5v.us-east-1.vpce.amazonaws.com:80 -vvv
```

Comment:
Are any of the numbers in the example for a specific project or something we would not want to publish? Also, if they are, might be more enlightening to know where the strings come from?

Reply:
> Are any of the numbers in the example for a specific project or something we would not want to publish? Also, if they are, might be more enlightening to know where the strings come from?

Those strings are something else!  I'm not sure I understand your exact question, but here's an attempt:

I replaced the IDs in the VPC service endpoint URL with randomly generated strings. 

In the command template immediately above:  I put placeholders for <endpoint-service-dns-url> and <port>.